src/lib/edge-functions/registry.cjs

```diff
@@ -285,10 +285,15 @@ class EdgeFunctionsRegistry {
       declarations,
       functions: this.functions,
     })
-    const routes = manifest.routes.map((route) => ({
+    const preCacheRoutes = manifest.routes.map((route) => ({
       ...route,
       pattern: new RegExp(route.pattern),
     }))
+    const postCacheRoutes = manifest.post_cache_routes.map((route) => ({
```

Comment:
I think you could merge the two routes arrays and have a single map?

```js
const routes = [...manifest.routes, ...manifest.post_cache_routes].map((route => ({
  ...route,
  pattern: new RegExp(route.pattern),
}))
```

Reply:
Better! Thanks, done!